content/v2.0/reference/flux/stdlib/contrib/influxdb/select.md

```diff
@@ -0,0 +1,180 @@
+---
+title: influxdb.select() function
+description: >
+  The `influxdb.select()` function is an alternate implementation of `from()`, `range()`, `filter()`
+  and `pivot()` that returns pivoted query results and masks the `_start` and `_stop` column
+  Results are similar to those returned by InfluxQL `SELECT` statements.
+menu:
+  v2_0_ref:
+    name: influxdb.select
+    parent: contrib_influxdb
+weight: 202
+v2.0/tags: [functions, package, query]
+related:
+  - /v2.0/reference/flux/stdlib/built-in/inputs/from/
+  - /v2.0/reference/flux/stdlib/built-in/transformations/range/
+  - /v2.0/reference/flux/stdlib/built-in/transformations/filter/
+  - /v2.0/reference/flux/stdlib/built-in/transformations/pivot/
+---
+
+The `influxdb.select()` function is an alternate implementation of `from()`, `range()`, `filter()`
+and `pivot()` that returns pivoted query results and masks the `_start` and `_stop` columns.
+Results are similar to those returned by InfluxQL `SELECT` statements.
+
+_**Function type:** Input_
+
+```js
+import "contrib/jsternberg/influxdb"
+
+influxdb.select(
+  from: "example-bucket",
+  start: -1d,
+  stop: now(),
+  m: "example-measurement",
+  fields: [],
+  where: (r) => true,
+  host: "https://example.com",
+  org: "example-org",
+  token: "MySuP3rSecr3Tt0k3n"
+)
+```
+
+## Parameters
+
+{{% note %}}
+[host](#host), [org](#org), and [token](#token) parameters are only required when
+querying data from a **different organization** or a **remote InfluxDB instance**.
+{{% /note %}}
+
+### from
+<span class="req">Required</span> Name of the bucket to query.
+
+_**Data type:** String_
+
+### start
+<span class="req">Required</span> Earliest time to include in results.
+Results **include** points that match the specified start time.
+Use a relative duration or absolute time.
+For example, `-1h` or `2019-08-28T22:00:00Z`.
+Durations are relative to `now()`.
+
+_**Data type:** Duration | Time_
+
+### stop
+Latest time to include in results.
+Results **exclude** points that match the specified stop time.
+Use a relative duration or absolute time.
+For example, `-1h` or `2019-08-28T22:00:00Z`.
+Durations are relative to `now()`.
+Defaults to `now()`.
+
+_**Data type:** Duration | Time_
+
+### m
+<span class="req">Required</span> Name of the measurement to query.
+
+_**Data type:** String_
+
+### fields
+List of fields to query.
+
+_**Data type:** Array of Strings_
+
+### where
+A single argument predicate function that evaluates true or false and filters results based on tag values.
+Records are passed to the function.
```

Comment:
> Records are passed to the function before the pivot.

At least some kind of reference to that so people don't try to do `r.usage_user` and expect it to work.